web/models/package.ex

```diff
@@ -122,8 +122,7 @@ defmodule HexWeb.Package do
   def all_with_docs do
     from(p in Package,
          order_by: p.name,
-         where: not is_nil(p.docs_updated_at),
-         select: {p.name, p.docs_updated_at})
+         where: not is_nil(p.docs_updated_at))
```

Comment:
okay, I can revert this change, it's just I saw that hex.pm/sitemap.xml isn't doing a select and is fetching even more packages. I'll add this optimization to that controller along with ordering by name I guess since we do it here as well